### PR TITLE
Remove unnecessary use of 'Option Strict Off'

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/EditorFeatures/Test2/Diagnostics/AbstractCrossLanguageUserDiagnosticTest.vb [hunk 1]
```diff
@@ -61,7 +56,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics
                 onAfterWorkspaceCreated?.Invoke(workspace)
 
                 Dim diagnosticAndFix = Await GetDiagnosticAndFixAsync(workspace)
-                Dim codeActions = diagnosticAndFix.Item2.Fixes.Select(Function(f) f.Action).ToList()
+                Dim codeActions As IList(Of CodeAction) = diagnosticAndFix.Item2.Fixes.Select(Function(f) f.Action).ToList()
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Test2/Diagnostics/AbstractCrossLanguageUserDiagnosticTest.vb:59]: surprising.  why is this needed?

File: src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/DynamicViewTests.vb [hunk 2]
```diff
@@ -2,6 +2,7 @@
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
 
+' These tests specifically cover late binding scenarios
```
Comment (CyrusNajmabadi) [inline src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/DynamicViewTests.vb:5]: i'm surprised the *tests* need this though, as opposed to the test having strings in them that have VB code in them that use late binding.

File: src/EditorFeatures/Test2/Diagnostics/AbstractCrossLanguageUserDiagnosticTest.vb (hunk 1 above)
Comment (sharwell) [reply src/EditorFeatures/Test2/Diagnostics/AbstractCrossLanguageUserDiagnosticTest.vb:59]: The line right after assigns `IList(Of CodeAction)`

File: src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/DynamicViewTests.vb (hunk 2 above)
Comment (sharwell) [reply src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/DynamicViewTests.vb:5]: The tests use `ExpandoObject`

**Comment by RikkiGibson** [comment]
@sharwell we can merge this if it's ready.

**Comment by RikkiGibson** [comment]
@sharwell Sorry that I did not merge any of the PRs yesterday that I left these comments on. I think adding a reaction to my comment did not send me a notification.

**Comment by sharwell** [comment]
@RikkiGibson it's no problem. Just as easy if I do it.
